gap/DocumentationTree.gi

```diff
@@ -541,7 +541,7 @@ InstallMethod( WriteDocumentation, [ IsString, IsStream ],
     ## In case the list is empty, do nothing.
     ## Once the empty string = empty list bug is fixed,
```

Comment:
Wait, what bug is that? Does it have an open issue in this repo? If not, perhaps add one? And then reference the number of that issue in that comment?

Reply:
In GAP, it is
```
gap> [ ] = "";
true
```
I do not consider this a bug anymore, so this comment should be updated to something like
> Since it is `[ ] = ""` ...
However, should this be done for this PR? The comment is old.

Reply:
It can be done in this PR or in a new one; I simply stumbled over this, trying to understand what is going on in this PR (which I still don't really know, but that's OK :-).